### ISSUE TITLE
WebR issue: unable to load shared object

### ISSUE THREAD

**Issue by isbool** [s-u/fastmatch#13]
Hey, im trying to use a custom compiled version of phangorn in webR and i get the following error:

An error occurred: package or namespace load failed for ‘phangorn’ in dyn.load(file, DLLpath = DLLpath, ...):
webr.js?v=2d081c24:2588  unable to load shared object '/usr/lib/R/library/fastmatch/libs/fastmatch.so':
webr.js?v=2d081c24:2588   Could not load dynamic lib: /usr/lib/R/library/fastmatch/libs/fastmatch.so
webr.js?v=2d081c24:2588 LinkError: WebAssembly.Instance(): Import #28 "env" "R_registerRoutines": imported function does not match the expected type

**Comment by isbool** [comment]
You can solve this by commenting out the src/dummy.c

```
// extern void R_registerRoutines(void);
// extern void R_useDynamicSymbols(void);

// void dummy(void) {
//     R_registerRoutines();
//     R_useDynamicSymbols();
// }
```

then it compiles

**Comment by s-u** [comment]
Yes, simply remove `src/dummy.c` as its only purpose is to appease CRAN R CMD checks which don't recognize `NAMESPACE ` symbol registrations - it is dead code.
